doc/tutorials/outline.md

```diff
@@ -0,0 +1,64 @@
+# Tutorial Outline
+
+These guides already exists
+
+- [components](components.md)
+- [development](development.md)
+- [interactivity](interactivity.md)
+- [layouts](layouts.md)
+- [param](param.md)
+- [structure](structure.md)
+- [styling](styling.md)
+
+They should be refactored into the below
+
+## Beginner: Most important, basic stuff to learn
+
+- [ ] Getting installed (just a link to getting started)
+  - [ ] Use a cloud development environment: Binder or Github spaces
+  - [ ] Use panels documentation in your browser: Panelite
```

Comment:
Are these beginner?

Reply:
You are not a Python beginner :-) The purpose is to make it easy for beginners. I've seen very often both locally and at conference events that people spend all their time trying to get installed. Can we remove that friction?

I will push an updated formulating like below. Feel free to comment again.

- [ ] Using a pre-installed environment for this tutorial: Purpose is to lower Barrier of entry. Make it possible to do at large conference with poor internet. Use Binder, Github spaces, Anaconda Notebooks, Panelite ???